src/ncurses_ui.cc

```diff
@@ -1228,6 +1256,14 @@ void NCursesUI::set_ui_options(const Options& options)
         m_wheel_down_button = wheel_down_it != options.end() ?
             str_to_int_ifp(wheel_down_it->value).value_or(5) : 5;
     }
+
+    {
+        auto it = options.find("ncurses_menu_layout"_sv);
+        if (it == options.end() or it->value == "horizontal")
+            m_menu_vertical = false;
+        else if (it->value == "vertical")
+            m_menu_vertical = true;
```

Comment:
Those lines can be simplified into

```cpp
m_menu_vertical = it != options.end() and it->value == "vertical";
```

Reply:
There's a slight difference, if it was set to something other than "horizontal" or "vertical" the value isn't changed with the current code.

Reply:
I understand, but since no errors are shown anyway, it's simpler to treat any value that doesn't set the boolean to `true` as `false`. That's how the other options are handled as well.

Reply:
That makes sense, I've implemented the change.